premailer/premailer.py

```diff
@@ -417,6 +417,12 @@ def transform(self, pretty_print=True, **kwargs):
                         parent.attrib[attr].startswith('cid:')
                     ):
                         continue
+                    if parent.attrib[attr].startswith("//"):
```

Comment:
This is already supported if you set
```transform('...', base_url='http://www.peterbe.com/')```

https://github.com/peterbe/premailer#turning-relative-urls-into-absolute-urls